app/controllers/dashboard_controller.rb

```diff
@@ -14,4 +16,22 @@ def index
       end
     end
   end
+
+  def kubectl_config
```

Comment:
Yes, it for sure should be refactorized. Problem is that this is `kubernetes` related, but it gets all the information from `salt`, so yeah...